bodhi-server/bodhi/server/models.py

```diff
@@ -213,6 +213,8 @@ def db_type(cls):
 class DeclEnumType(SchemaType, TypeDecorator):
     """A database column type for an enum."""
 
+    cache_ok = True
```

Comment:
I'll address this in PR #4437, could you remove it from here please?

Reply:
whoops! sorry! that snuck in there....

removed now